CLI env update cannot set a variable value to an empty string

**Describe the bug**

`mcp-use servers env update <KEY> --server <id> --value ""` is rejected as if no update fields were provided. This prevents users from intentionally clearing an environment variable value while keeping the variable itself.

The issue is in `updateEnvCommand`: the no-op guard uses `!options.value`, so Commander's empty string value is treated the same as an omitted `--value`. Later code correctly checks `options.value !== undefined`, but it is never reached for empty strings.

**To Reproduce**

1. Run `mcp-use servers env update API_KEY --server <server-id> --value ""`.
2. The CLI prints `Nothing to update. Provide at least one of --value, --env, --sensitive.`
3. The API update call is not made.

Focused local repro used for verification:

```ts
it("allows updating an env var value to an empty string", async () => {
  const { createEnvCommand } = await import("../src/commands/env.js");
  mockApiInstance.listEnvVariables.mockResolvedValue([
    {
      id: VAR_ID,
      key: "API_KEY",
      value: "old",
      environments: ["production"],
      branch: null,
      sensitive: false,
    },
  ]);
  mockApiInstance.updateEnvVariable.mockResolvedValue({
    id: VAR_ID,
    key: "API_KEY",
    value: "",
    environments: ["production"],
    branch: null,
    sensitive: false,
  });

  const cmd = createEnvCommand();
  await cmd.parseAsync(
    ["update", "API_KEY", "--server", SERVER_ID, "--value", ""],
    { from: "user" }
  );

  expect(mockApiInstance.updateEnvVariable).toHaveBeenCalledWith(
    SERVER_ID,
    VAR_ID,
    expect.objectContaining({ value: "" })
  );
});
```

**Expected behavior**

Passing `--value ""` should be accepted and should call `updateEnvVariable(..., { value: "" })`. Only `options.value === undefined` should mean the flag was omitted.

**Additional context / verification**

I verified this locally by adding the focused test above to `libraries/typescript/packages/cli/tests/servers-update-env-branch.test.ts` and running the CLI package tests. The relevant failure is:

```text
✗ Nothing to update. Provide at least one of --value, --env, --sensitive.
AssertionError: expected updateEnvVariable to be called ...
Number of calls: 0
```

The same test should pass if the no-op guard checks `options.value === undefined` instead of using truthiness.

--- a/.github/workflows/bump-mcp-use-reusable.yml
+++ b/.github/workflows/bump-mcp-use-reusable.yml
@@ -61,7 +61,7 @@ jobs:
 
           if [ "${{ steps.pm.outputs.pm }}" = "pnpm" ]; then
             corepack enable
-            pnpm install --no-frozen-lockfile --config.dangerously-allow-all-builds=true
+            pnpm install --no-frozen-lockfile
             pnpm run build
           else
             npm install

--- a/libraries/typescript/.changeset/empty-env-value-update.md
+++ b/libraries/typescript/.changeset/empty-env-value-update.md
@@ -1,0 +1,5 @@
+---
+"@mcp-use/cli": patch
+---
+
+Allow updating CLI environment variable values to an empty string.

--- a/libraries/typescript/packages/cli/src/commands/env.ts
+++ b/libraries/typescript/packages/cli/src/commands/env.ts
@@ -215,7 +215,11 @@ async function updateEnvCommand(
   try {
     await requireLogin();
 
-    if (!options.value && !options.env && options.sensitive === undefined) {
+    if (
+      options.value === undefined &&
+      !options.env &&
+      options.sensitive === undefined
+    ) {
       console.error(
         chalk.red(
           "✗ Nothing to update. Provide at least one of --value, --env, --sensitive."

--- a/libraries/typescript/packages/cli/tests/servers-update-env-branch.test.ts
+++ b/libraries/typescript/packages/cli/tests/servers-update-env-branch.test.ts
@@ -230,6 +230,40 @@ describe("env-var branch scoping + resolve-by-key", () => {
     );
   });
 
+  it("allows updating an env var value to an empty string", async () => {
+    const { createEnvCommand } = await import("../src/commands/env.js");
+    mockApiInstance.listEnvVariables.mockResolvedValue([
+      {
+        id: VAR_ID,
+        key: "API_KEY",
+        value: "old",
+        environments: ["production"],
+        branch: null,
+        sensitive: false,
+      },
+    ]);
+    mockApiInstance.updateEnvVariable.mockResolvedValue({
+      id: VAR_ID,
+      key: "API_KEY",
+      value: "",
+      environments: ["production"],
+      branch: null,
+      sensitive: false,
+    });
+
+    const cmd = createEnvCommand();
+    await cmd.parseAsync(
+      ["update", "API_KEY", "--server", SERVER_ID, "--value", ""],
+      { from: "user" }
+    );
+
+    expect(mockApiInstance.updateEnvVariable).toHaveBeenCalledWith(
+      SERVER_ID,
+      VAR_ID,
+      expect.objectContaining({ value: "" })
+    );
+  });
+
   it("remove by UUID skips the key lookup", async () => {
     const { createEnvCommand } = await import("../src/commands/env.js");
     mockApiInstance.deleteEnvVariable.mockResolvedValue(undefined);

--- a/lychee.toml
+++ b/lychee.toml
@@ -21,6 +21,10 @@ exclude = [
   
   # Lychee GitHub repository - self-reference
   "^https://github\\.com/lycheeverse/lychee",
+
+  # Langfuse Cloud API endpoints - 403s from bot protection in CI, valid defaults
+  "^https://(us\\.|jp\\.|hipaa\\.)?cloud\\.langfuse\\.com",
+  "cloud\\.langfuse\\.com",
   
   # Example/template URLs
   "^https://github\\.com/your-org/",

